### ISSUE TITLE
Can't deploy to testnet with cairo v2.1.0

### ISSUE THREAD

**Issue by credence0x** [xJonathanLEI/starkli#21]
Running `$starkli declare ./target/dev/x.sierra.json --compiler-version 2.1.0`

returns 
```
error: invalid value '2.1.0' for '--compiler-version <COMPILER_VERSION>'
 [possible values: 2.0.1]

 tip: a similar value exists: '2.0.1'
```

**Comment by xJonathanLEI** [comment]
Starkli needs an upgrade here. Note that normally you wouldn't need to manually set the compiler version. Starkli automatically detects it for you.

**Comment by credence0x** [comment]
Yeah. The issue was kind of a reminder for an update to be pushed. Unnecessary?

**Comment by xJonathanLEI** [comment]
Definitely not unnecessary. Thanks for flagging the issue! Working on a release now.

**Comment by xJonathanLEI** [comment]
Compiler v2.1.0 added and used by default for testnets and the integration environment. Released in version [0.1.9](https://github.com/xJonathanLEI/starkli/releases/tag/v0.1.9).

Just run `starkliup` and retry. It should work now (even without the `--compiler-version` flag).

**Comment by credence0x** [comment]
🫡
